notebooks/network_analysis_sec.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 600x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(6, 6))\n",
+    "plt.pie(heating_sums)\n",
+    "plt.title('Heating Supply Distribution')\n",
+    "plt.legend(heating_types, loc='upper left', bbox_to_anchor=(1, 0.8))\n",
+    "plt.axis('equal')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Electricity Sector \n",
+    "The share of different carriers (capacity and energy)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "           carrier           \n",
+       "Generator  Coal                   500.000000\n",
+       "           Combined-Cycle Gas    1838.950905\n",
+       "           Offshore Wind (AC)       0.000000\n",
+       "           Offshore Wind (DC)       0.000000\n",
+       "           Oil                    150.066350\n",
+       "           Onshore Wind           784.000000\n",
+       "           Open-Cycle Gas         543.978622\n",
+       "           Run of River           398.225382\n",
+       "           Solar                  142.074179\n",
+       "Name: p_nom, dtype: float64"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "capacity_stats = n.statistics.installed_capacity(comps=[\"Generator\"])\n",
```

Comment:
Note, we also have the technology "Reservoir & Dams" which are modelled as StorageUnits and not Generators. You can get them by `n.statistics.installed_capacity(comps=["StorageUnit"])` and please include it here as well

Reply:
This also applies for the energy analysis further down the noteboook